IEEE Xplore.js

```diff
@@ -2,18 +2,18 @@
 	"translatorID": "92d4ed84-8d0-4d3c-941f-d4b9124cfbb",
 	"label": "IEEE Xplore",
 	"creator": "Simon Kornblith, Michael Berkowitz, Bastian Koenings, and Avram Lyon",
-	"target": "^https?://([^/]+\\.)?ieeexplore\\.ieee\\.org/([^#]+[&?]arnumber=\\d+|(abstract/)?document/|search/(searchresult|selected)\\.jsp|xpl/(mostRecentIssue|tocresult)\\.jsp\\?)",
+	"target": "^https?://([^/]+\\.)?ieeexplore\\.ieee\\.org/([^#]+[&?]arnumber=\\d+|(abstract/)?document/|search/(searchresult|selected)\\.jsp|xpl/((mostRecentIssue|tocresult)\\.jsp\\?|conhome/\\d+/proceeding))",
```

Comment:
With the new parenthesis this is quite hard to understand at first. Can we change it to easier expression by repeating the `xpl/`, i.e. just adding `|xpl/conhome/\\d+/proceeding` at the end?

Reply:
Thanks for your suggestion. I have changed it as you suggested.